feat(trace-view): Add documentation for the new Trace View (#3347)
getsentry/sentry-docs: src/docs/product/performance/event-detail.mdx

#### With Transaction Summary

Select the [Performance Homepage](/product/performance/), then click the affected transaction to display the transaction data.

#### Querying for this Transaction

Scroll down to the "Trace Details" context panel in either the Issue Details or the Discover Event Details page, and click on the "View Summary" button. This will maintain the context of the current Sentry event.

_Note_: Users on the Team or Business plans can also view a list of transaction events by clicking on the "Transactions" pre-built query in [Discover](/product/discover-queries/) or by performing a search with the `event.type:transaction` condition the Discover Query Builder view.

> ### Quick Trace

Reviewer: see previous edits

The quick trace (which displays top right of the span view) is an abbreviated view of the related [trace](/product/performance/distributed-tracing) for the current transaction.
The quick trace displays up to six nodes, each representing different groups of the event's trace:
- **Root**: Starting transaction

> - **Ancestor**: Events that are descendants of the root, and are directly related to the current event

Reviewer 2: Nit: using the term descendants to explain ancestors seems a little ironic.

Author: Heh, what about
```suggestion
- **Ancestor**: Events in the generations between the Root and the Parent of the current Event
```

- **Parent**: Event that directly started the current event
- **This Event**: Node representing the current event being viewed
- **Children**: Events that the current event directly started

> - **Descendants**: Future generations of events created by children of this event

Reviewer 2: We don't use the term "generation" anywhere else from what I can see, do we want to use it here?

Author: Hmm we specifically call out that the parent-child thing is a metaphor in `/product/performance/distributed-tracing/` so I think using generations to keep with that makes sense to me
